components/cop_codegen/src/rpn_function.rs

```diff
@@ -75,10 +102,24 @@ impl parse::Parse for RpnFnAttr {
                 "`varg` and `raw_varg` conflicts to each other",
             ));
         }
+        if !is_varg && !is_raw_varg && min_args != None {
+            return Err(Error::new_spanned(
+                config_items,
+                "`min_args` is only available when `varg` or `raw_varg` presents",
+            ));
+        }
+        if validator.is_some() && min_args != None {
```

Comment:
I think the `min_args` and `validator` can coexist (Maybe we can append generated tokens to customized tokens?).

Reply:
Very good suggestion 👏